src/js/components/ServerErrorModal.js

```diff
@@ -56,7 +56,7 @@ module.exports = class ServerErrorModal extends mixin(StoreMixin) {
     });
   }
 
-  handleServerError(errorMessage) {
+  handleServerError({message:errorMessage}) {
```

Comment:
Hello @MatApple , this function does not forward the `details` array that might be needed in order to compile a more verbose response. For now I will use this function to the actions I introduced, but we might need to think about this.